packages/react/src/react-component-lib/slottedSVG.tsx

```diff
@@ -8,8 +8,9 @@ function slot(name = "") {
   return { ref: (e: any) => (e ? e.setAttribute("slot", name) : null) };
 }
 
-export const SlottedSVG: FC<any> = ({ props, path, slot: slotName }) => (
-  <svg {...slot(slotName)} {...props} {...defaultProps}>
-    <path d={path} />
-  </svg>
-);
+export const SlottedSVG: FC<any> = ({ path, slot: slotName, children, ...props}) => {
+  return (
+  <svg {...slot(slotName)} {...props} {...defaultProps} >
+    {!!path && (<path d={path} />)}
+    {children}
+  </svg>)};
```

Comment:
If you've removed the curly bracket from line 11, you won't need the bracket here anymore and the curved bracket can go on the line below.